acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationCachingIT.java

```diff
@@ -153,58 +155,75 @@ public void testPolicyEvalCacheWhenPolicySetChanges() throws Exception {
+        String policyFile2 = "src/test/resources/policies/deny-all.json";
+        String policyFile3 = "src/test/resources/policies/single-site-based.json";
         String endpoint = this.acsUrl;
-        String policyFile = "src/test/resources/policies/indeterminate.json";
-        testPolicyName = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
-                policyFile);
-        PolicyEvaluationRequestV1 policyEvaluationRequest = this.policyHelper.createEvalRequest(NOT_MATCHING_ACTION,
-                DEFAULT_SUBJECT_ID, DEFAULT_RESOURCE_IDENTIFIER, null);
+
+        this.privilegeHelper.putSubject(this.acsAdminRestTemplate, subject, endpoint, this.acsZone1Headers,
+                this.privilegeHelper.getDefaultAttribute());
+
+        List<PolicySet> policySets = new ArrayList<PolicySet>();
+        policySets.add(this.jsonUtils.deserializeFromFile("src/test/resources/policies/single-site-based.json",
+                PolicySet.class));
+        policySets
+                .add(this.jsonUtils.deserializeFromFile("src/test/resources/policies/deny-all.json", PolicySet.class));
+
+        String policySet1 = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
+                policyFile1);
+        String policySet2 = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
+                policyFile2);
+        
+        // test with a valid policy set evaluation order list
+        PolicyEvaluationRequestV1 policyEvaluationRequest = this.policyHelper.createMultiplePolicySetsEvalRequest(
+                MARISSA_V1.getSubjectIdentifier(), "sanramon", Arrays.asList(policySet1, policySet2));
+
         ResponseEntity<PolicyEvaluationResult> postForEntity = this.acsAdminRestTemplate.postForEntity(
                 endpoint + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
                 new HttpEntity<>(policyEvaluationRequest, this.acsZone1Headers), PolicyEvaluationResult.class);
+
         Assert.assertEquals(postForEntity.getStatusCode(), HttpStatus.OK);
         PolicyEvaluationResult responseBody = postForEntity.getBody();
-        Assert.assertEquals(responseBody.getEffect(), Effect.NOT_APPLICABLE);
-        this.policyHelper.deletePolicySet(this.acsAdminRestTemplate, this.zoneHelper.getZone1Url(), testPolicyName,
-                this.acsZone1Headers);
+        Assert.assertEquals(responseBody.getEffect(), Effect.DENY);
 
-        policyFile = "src/test/resources/policies/permit-admin-only-using-condition.json";
-        testPolicyName = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
-                policyFile);
+        // test with one of the policy sets changed from the evaluation order list
+        String policySet3 = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
```

Comment:
I know that this is a test but can we pick a variable name that provides some context to the test code e.g. "expectPermitPolicySet" or similar?